### PR TITLE
golangci-lint: Add config file

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds a config file that configures `make lint` to use the linters needed for this project

### CODE REVIEW

**Review by orelmisan — COMMENTED** [review]
Could you please add a link to the original configuration from `kiagnose/kiagnose`?

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ GO_MOD_VERSION=$(shell hack/go-mod-version.sh)
 
 all: check build
 
-check: fmt check-uncommitted lint test/unit
+check: lint test/unit
```
Comment (orelmisan) [inline Makefile:17]: They `fmt` and `check-uncommitted` were not removed.
The commit message is unclear.

File: Makefile [hunk 2]
```diff
@@ -52,7 +52,7 @@ fmt:
 	$(CRI_BIN) run --rm \
 	           --volume `pwd`:$(CURDIR):Z \
 	           --workdir $(CURDIR) \
-	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) gofmt -w .
+	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) gofmt -w -s .
```
Comment (orelmisan) [inline Makefile:55]: Could you please add an explanation to the commit message about what this flag does and why is it needed?

Comment (RamLavi) [reply Makefile:55]: when I did, I realized I don't want this flag. will remove it from here and from linter

**Review by orelmisan — COMMENTED** [review]
[Second](https://github.com/kiagnose/kubevirt-dpdk-checkup/pull/8/commits/3fd3aa88c7901520508b3d017c1a314fe06d3712) commit message:
> now that linter was added with gofmt check - it is redundant.Removing
from all makefile target

It was removed from the `check` target.

**Comment by RamLavi** [comment]
> [Second](https://github.com/kiagnose/kubevirt-dpdk-checkup/pull/8/commits/3fd3aa88c7901520508b3d017c1a314fe06d3712) commit message:
> 
> > now that linter was added with gofmt check - it is redundant.Removing
> > from all makefile target
> 
> It was removed from the `check` target.

fixed

**Review by orelmisan — APPROVED** [review]

**Pull request merged**